### DISCUSSION TITLE
Improved Azure Support

### DISCUSSION THREAD

**Discussion by xaevik** [imgproxy/imgproxy#1051] [General]
Just out of curiosity, is there any timeline as to when PR #1042 will make it into a release?

Right now we're manually applying the patch in our build pipelines.

**Comment by DarthSim** [comment]
Hi @xaevik. That PR doesn't contain any changes. Seems like a bot on your side force-pushed master branch to your fork

**Reply by xaevik** [reply]
Yeah it appears that is exactly what happened. I'll trace that down and submit a new PR. I apologize about that.

**Reply by xaevik** [reply]
Hi @DarthSim so the issue was traced to an old GitHub app I had installed called Pull (this was installed prior to when GitHub added the ability to sync forks). I removed the application, recovered the changeset and re-submitted a new PR. I apologize for the mess.
